test/cpp/end2end/xds/xds_fault_injection_end2end_test.cc

```diff
@@ -204,7 +204,13 @@ TEST_P(FaultInjectionTest, XdsFaultInjectionPercentageAbortViaHeaders) {
 
 TEST_P(FaultInjectionTest, XdsFaultInjectionPercentageDelay) {
   CreateAndStartBackends(1);
+  const uint32_t kFixedDelayMilliseconds = 100000;
+#ifndef NDEBUG
```

Comment:
I'm really not keen on putting `#ifndef`s in the middle of individual tests.  Can we not just increase the timeout even for opt builds?

Reply:
Done